File: backend/src/main/java/com/example/backend/models/Jelo.java
package com.example.backend.models;

public class Jelo {
    private int id;
    private int id_res;
    private String naziv_jela;
    private String sastojci;
    private int cena;
    public Jelo(int id, int id_res, String naziv_jela, String sastojci, int cena) {
        this.id = id;
        this.id_res = id_res;
        this.naziv_jela = naziv_jela;
        this.sastojci = sastojci;
        this.cena = cena;
    }
    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public int getId_res() {
        return id_res;
    }
    public void setId_res(int id_res) {
        this.id_res = id_res;
    }
    public String getNaziv_jela() {
        return naziv_jela;
    }
    public void setNaziv_jela(String naziv_jela) {
        this.naziv_jela = naziv_jela;
    }
    public String getSastojci() {
        return sastojci;
    }
    public void setSastojci(String sastojci) {
        this.sastojci = sastojci;
    }
    public int getCena() {
        return cena;
    }
    public void setCena(int cena) {
        this.cena = cena;
    }
    
}

File: models/Jelo.ts
export class Jelo {

    id: number=0;
    id_res: number=0;
    naziv_jela: string="";
    sastojci: string="";
    cena: number=0;
    kolicina: number=0;

    constructor(i: number, ir: number, nj: string, sas: string, cen: number) {
        this.id=i; this.id_res=ir; this.naziv_jela=nj; this.sastojci=sas; this.cena=cen;
    }
}